package.json

```diff
@@ -14,17 +14,12 @@
     "eslint": "^5.4.0",
     "eslint-config-chartjs": "^0.1.0",
     "gulp": "3.9.x",
-    "gulp-concat": "^2.6.1",
     "gulp-eslint": "^5.0.0",
-    "gulp-insert": "^0.5.0",
-    "gulp-replace": "^1.0.0",
-    "gulp-streamify": "^1.0.2",
-    "gulp-uglify": "^1.5.3",
     "gulp-util": "^3.0.8",
     "inquirer": "^6.2.0",
-    "merge-stream": "^1.0.1",
-    "semver": "^5.5.1",
-    "vinyl-source-stream": "^2.0.0"
+    "rollup": "^1.1.2",
```

Comment:
I believe you can remove the `browserify` dependency now

Reply:
Right! Done.